### PR TITLE
Encryption Codeunit

### PR DESCRIPTION
**Pull request by Gallimathias**
I have merged the two code units of @pzentner and @cvadstrup

### CODE REVIEW

File: Codeunits/Codeunit_50000.txt [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      hashBytes@50002 : DotNet "'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Array";
+      byte@50003 : DotNet "'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Byte";
+      stringBuilder@50004 : DotNet "'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Text.StringBuilder";
+      Encoding@50005 : DotNet "'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Text.Encoding";
+      i@50006 : Integer;
+    BEGIN
+      IF SourceString = '' THEN
+        EXIT;
+
+      //--- Using BEGIN
```
Comment (BardurKnudsen) [inline Codeunits/Codeunit_50000.txt]: Please do not use comments  -unless they are necessary to explain what goes on.

Comment (Gallimathias) [reply Codeunits/Codeunit_50000.txt]: This is exactly what these comments do, because they explain why I dispose and therefore mark the security area.

**Comment by Gallimathias** [comment]
@BardurKnudsen i have remove the comments

**Comment by BardurKnudsen** [comment]
We believe we have covered the proposed functionality by the additions to codeunit 1266.

**Comment by JesperSchulz** [comment]
The new library functions will ship with our November update. Thank you again for your contribution!

**Pull request closed without merging**